menu/MainFrame.cpp

```diff
@@ -178,20 +178,27 @@ void Func_Credits()
 #ifdef HW_RVL
   int iosversion = IOS_GetVersion();
   int isWiiVC = SYS_GetCoreMultiplier() >= 5.0;
-  sprintf(CreditsInfo,"Wii64 Beta 1.3 - IOS %i %s\n",iosversion,isWiiVC ? "WiiVC!":"");
+#ifdef GLN64_GX
+  sprintf(CreditsInfo,"Wii64 Beta 1.3 glN64 GFX - IOS %i %s\n",iosversion,isWiiVC ? "WiiVC!":"");
 #else
-	strcat(CreditsInfo,"Cube64 Beta 1.3\n");
-#endif
+  sprintf(CreditsInfo,"Wii64 Beta 1.3 Rice GFX - IOS %i %s\n",iosversion,isWiiVC ? "WiiVC!":"");
+#endif // GLN64_GX (Wii)
+#else
+#ifdef GLN64_GX
+	strcat(CreditsInfo,"Cube64 Beta 1.3 glN64 GFX\n");
+#else
+	strcat(CreditsInfo,"Cube64 Beta 1.3 Rice GFX\n");
+#endif // GLN64_GX (GameCube)
+#endif // HW_RVL
+	strcat(CreditsInfo,"WiiFlow MOD Beta5 by saulfabreg\n");

```

Comment:
Can you please revert this section if it's to be merged and I'll just update the credits later. Thanks.

Reply:
Sorry there @emukidid since i'm making also my unofficial WiiFlow MOD i am keeping working in it with my own builds, until i find a way for revert these changes.
I will make a new branch instead and put all my commits for this, and revert those others i've added and i will start another PR.

Reply:
Closing this PR, I will send a new PR for this with my changes on Credits reverted on my new branch "wii64-wiiflow-mod-PR".